README.md

```diff
@@ -7,6 +7,29 @@ Official bootstrap for running your own [Sentry](https://sentry.io/) with [Docke
  * Docker 1.10.0+
  * Compose 1.6.0+ _(optional)_
 
+## Up and Running
+
+Assuming you've just cloned this repository, the following steps 
+will get you up and running in no time!
+
+1. `mkdir -p data/{sentry,postgres}` - Make our local database and sentry config directories.
+    This directory is bind-mounted with postgres so you don't lose state!
+2. `docker-compose run --rm web config generate-secret-key` - Generate a secret key.
+    Add it to `docker-compose.yml` in `base` as `SENTRY_SECRET_KEY`.
+3. `docker-compose run --rm web upgrade` - Build the database.
+    Use the interactive prompts to create a user account.
+4. `docker-compose up -d` - Lift all services (detached/background mode).
+5. Access your instance at `localhost:9000`!
+
+Note that as long as you have your database bind-mounted, you should
+be fine stopping and removing the containers without worry.
+
+## Securing Sentry with SSL/TLS
+
+If you'd like to protect your Sentry install with SSL/TLS, there are
+fantastic SSL/TLS proxies like [HAProxy](http://www.haproxy.org/) 
+and [Nginx](https://www.nginx.com/).
```

Comment:
Link to `nginx.org` plz. nginx.com is the commercial entity, and nginx.org is the open source.